NETPROGRAM/BasicExercise/Exercise11.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+using System.Collections.Generic;
+
+namespace NETPROGRAM.BasicExercise
+{
+    public class Exercise11
+    {
+
+        public int BinaryNumberReverse(int n)
+        {
+            var number = 0;
+            var listNumber = new List<int>();
+
+            if(n == 0)
+            {
+                return 0;
+            }
+
+            while(n > 0)
+            {
+                listNumber.Add(n % 2);
+                n /= 2;
+            }
+
+            for(int i = 0; i<listNumber.Count; i++)
```

Comment:
Bài này chỉ thao tác trên bit, không dùng list

Reply:
done. Em xem thử đúng ý chưa.

Reply:
Chưa đúng, cố gắng suy nghĩ tiếp

Reply:
Khi nào mà anh dùng toàn phép toán trên bit (không có +-*/ ... )